packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -302,8 +318,31 @@ export const createDefaultCallingHandlers = memoizeOne(
       await call?.removeParticipant(fromFlatCommunicationIdentifier(userId));
     };
 
+    /* @conditional-compile-remove(PSTN-calls) */
+    const onAddParticipant = async (
+      participant:
+        | CommunicationUserIdentifier
+        | PhoneNumberIdentifier
+        | UnknownIdentifier
+        | MicrosoftTeamsUserIdentifier,
+      options?: AddPhoneNumberOptions
+    ): Promise<void> => {
+      const newACSorTeamsUser = participant as CommunicationUserIdentifier | MicrosoftTeamsUserIdentifier;
+      const newPSTNUser = participant as PhoneNumberIdentifier;
+      if (newPSTNUser) {
```

Comment:
I would simplify all this logic by doing this:
```tsx
if (participant.phoneNumber) {
  await call?.addParticipant(participant as PhoneNumberIdentifier, options);    
} else {
  await call?.addParticipant(participant as CommunicationUserIdentifier | MicrosoftTeamsUserIdentifier);
}
```

Reply:
OR use the `isPhoneNumberIdentifier` function exported by the communication-common package.
```tsx
import { isPhoneNumberIdentifier } from "@azure/communication-common";

if (isPhoneNumberIdentifier(participant)) {
  await call?.addParticipant(participant as PhoneNumberIdentifier, options);
} else {
  await call?.addParticipant(
    participant as CommunicationUserIdentifier | MicrosoftTeamsUserIdentifier
  );
}
```

Reply:
Second option is better as it used the intended methods.

Reply:
oh cool! will update this